uiowa/uiowa
# Feature: 175 footer logo work

# How to test

- Pull down locally or skip to Percy review option listed below.
- Verify that footer logo is completely changed and the new logo links to new 175 website
- **Note:** build is failing because of Percy visual change, please review and approve visual changes in https://percy.io/ (contact me for browserstack access)
